src/app/welcome-page/welcome-page.component.html

```diff
@@ -50,7 +50,7 @@ <h1>
     <img class="section-img" src="/info_section_1_illustration.svg" />
 
     <div class="box">
-      <h2><span> Document the evidence,</span> unmask the trolls</h2>
+      <h2>Document the evidence, unmask the trolls</h2>
```

Comment:
Damn, we're moving away from 2 level-heading style????

Reply:
Yes. They want it all uniform.